{{cookiecutter.project_slug}}/gulpfile.js

```diff
@@ -127,7 +127,13 @@ function initBrowserSync() {
         `${paths.js}/*.js`,
         `${paths.templates}/*.html`
       ], {
+        {%- if cookiecutter.use_docker == 'n' %}
         proxy: "localhost:8000"
+        {% else %}
+        proxy:  "django:8000",
+        // Doesn't work from inside the Docker image
```

Comment:
What does this mean, out of curiosity? What about this doesn't work, exactly?

Reply:
By default, browser sync tries to open the default browser at the location it runs. It works great when it's running directly on the same machine, but doesn't work from inside a container.

Will elaborate this comment.